lib/ssz.ex

```diff
@@ -84,6 +84,8 @@ defmodule Ssz do
 
   # Ssz types can have special decoding rules defined in their optional encode/1 function,
   defp encode(%name{} = struct) do
+    Code.ensure_loaded!(name)
```

Comment:
As we always use function_exported paired with ensure_loaded, I'd create a small function to do both:

```elixir
def exported?(module, function, arity) do
  Code.ensure_loaded!(name)
  function_exported?(name, :encode, 1)
end
```

and then call that one directly.

Reply:
Good idea!